circle.yml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  pre:
+    - git config --global user.email "ci@codeclimate.com"
+    - git config --global user.name "Code Climate CI"
```

Comment:
We run some git commands in the specs and [they error without an identity](https://circleci.com/gh/codeclimate/ruby-test-reporter/1)